README.md

```diff
@@ -6,6 +6,8 @@
 
 ExDoc is a tool to generate documentation for your Elixir projects. In case you are looking for documentation for Elixir itself, [check out Elixir's website][elixir-lang].
 
+For documentation on writing documentation [check out Elixir's documentation][hex-writing-docs].
```

Comment:
sorry for some reason I thought it this PR was submitted to Elixir.

how about:
> To learn about how to document your projects check out [Elixir's writing documentation page][hex-writing-docs]